src/otx/v2/adapters/datumaro/adapter/action_dataset_adapter.py

```diff
@@ -118,9 +118,8 @@ class ActionClassificationDatasetAdapter(ActionBaseDatasetAdapter):
 
     def get_otx_dataset(self) -> DatasetEntity:
         """Convert DatumaroDataset to DatasetEntity for Acion Classification."""
-        label_information = self._prepare_label_information(self.dataset)
-        self.label_entities = label_information["label_entities"]
-
+        if not hasattr(self, "label_entities"):
+            super().get_label_schema()
```

Comment:
`get_label_schema` is always called prior to get dataset as following https://github.com/openvinotoolkit/training_extensions/blob/376ce9c155a4afe9e72baa87a6f6a693e3108939/src/otx/v2/api/core/dataset.py#L318. Accordingly, `label_entities` should be computed within `get_label_schema`. Could you check the order of function calls during this operation?

Reply:
Yeah, I also think that. However, some unit tests are not written in this order, that's the reason why I changed the unit test code for the detection. I think changing unit tests is the better way. Thanks for pointing out.


<img width="985" alt="image" src="https://github.com/openvinotoolkit/training_extensions/assets/80735344/1565bbd4-2911-49be-9e78-c54291b3e5c6">

Reply:
I also prefer removing this part

Reply:
Please make a commit for updating unit tests and attach the commit number here.

Reply:
Done. https://github.com/openvinotoolkit/training_extensions/pull/2631/commits/3ca5154cc9fd8172865590ad46d03db01b772a91